elytron-oidc-client/subsystem/src/main/resources/org/wildfly/extension/elytron/oidc/LocalDescriptions.properties

```diff
@@ -1,144 +1,142 @@
-elytron-oidc-client=WildFly Extension for Elytron OIDC
-elytron-oidc-client.add=Add the Elytron OIDC subsystem
-elytron-oidc-client.remove=Remove the Elytron OIDC subsystem
-elytron-oidc-client.realm=A Keycloak realm
-elytron-oidc-client.provider=An OpenID Connect provider
-elytron-oidc-client.secure-deployment=A deployment secured by an OpenID Connect provider
+elytron-oidc-client=WildFly Extension for Elytron OIDC.
+elytron-oidc-client.add=Add the Elytron OIDC subsystem.
+elytron-oidc-client.remove=Remove the Elytron OIDC subsystem.
+elytron-oidc-client.realm=Configuration for a Keycloak realm. This is provided for convenience. You can copy the configuration in the Keycloak client adapter and use it here. Using the provider attribute is recommended instead.
```

Comment:
I don't think the `realm` element config can be copied from the Keycloak Admin Console (only the configuration for a specific client could be copied).

Here, maybe we can update the description to something like:
Configuration for a Keycloak realm. If you have multiple deployments secured by the same realm you can share the realm configuration here.

Reply:
I believe this part was copied from here: https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/8-beta/html/using_single_sign-on_with_jboss_eap/ref_elytron-oidc-client-subsystem-attributes_default 

I'm not sure how accurate the description in the link is, but I've changed this one as advised.